frontend/src/components/material/MaterialCreateItem.vue

```diff
@@ -57,7 +57,7 @@
         </p>
         <v-btn :to="campRoute(camp, 'admin')">
           <v-icon :left="$vuetify.breakpoint.mdAndUp">mdi-cogs</v-icon>
-          {{ $tc('views.camp.navigationCamp.admin') }}
+          {{ $tc('components.camp.campSettings.title') }}
```

Comment:
Hier stimmt der Key noch nicht. Der Text auf diesem Button ist nicht zwingend derselbe wie im Titel der Settings-View. Der Button könnte auch mal "Hier Materiallisten erfassen" o.ä. heissen oder in einer anderen Sprache leicht anders heissen. Wenn jemand den Titel der CampSettings-Komponente ändert erwartet er nicht, dass das sonstwo noch Auswirkungen hat. Das System mit den pfadabhängigen Keys haben wir ja überhaupt erst geschaffen damit wir einen Überblick haben, wo jede Übersetzung verwendet wird.
Wiederverwenden können wir meiner Meinung nach nur die `entity` und `global` Übersetzungen.

Reply:
Habe ich angepasst.
Der war vorher schon nicht gut, wir sollten da wohl mal das ganze anschauen. Sonst tauchen immer wieder diese Probleme auf, die eigentlich nicht viel mit dem eigentlichen PR zu tun haben.

Reply:
Ja. Vielleicht schaffen wir es sogar, eine Linter Regel oder so für 95% dieser Translation Keys zu schreiben..?

Reply:
Habs aufgenommen: https://github.com/ecamp/ecamp3/issues/2932